pkg/scalers/solr_scaler.go

```diff
@@ -24,16 +23,42 @@ type solrScaler struct {
 }
 
 type solrMetadata struct {
-	host                       string
-	collection                 string
-	targetQueryValue           float64
-	activationTargetQueryValue float64
-	query                      string
-	triggerIndex               int
+	triggerIndex int
+
+	Host                       string  `keda:"name=host, order=triggerMetadata"`
+	Collection                 string  `keda:"name=collection, order=triggerMetadata, optional"`
```

Comment:
nit: based on the tests and `Validate()` method, looks like this is not `optional`
```suggestion
	Collection                 string  `keda:"name=collection, order=triggerMetadata"`
```